py_neuromodulation/nm_RMAP.py

```diff
@@ -351,8 +351,8 @@ def get_fingerprints_from_path_with_cond(
 
     @staticmethod
     def save_Nii(
-        fp: np.array,
-        affine: np.array,
+        fp: np.ndarray,
```

Comment:
Interesting, just figured out that `np.array` is a function and `np.ndarray` is a class. So thanks for catching! :)